data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
+  "origin": {
+    "date_human-readable": "mid-4th c. CE",
+    "date_normalized": "0325/0375",
+    "language": "Greek",
+    "place": {}
+  },
+  "genre": ["Theological Works", "Commentary, Biblical", "Commentary, Old Testament", "Commentary, Genesis"],
+  "authors": [
+    {
+      "id": "https://sinaimanuscripts.library.ucla.edu/person/55",
+      "role": "author"
+    }
+  ],
+  "associated_persons": [],
```

Comment:
e.g., for commemorated saints? In SDP do we have separate field for authors? Or just use a role field to specify authorship, correct/disputed attribution, etc.?